scripts/mysqld_safe.sh

```diff
@@ -21,9 +21,11 @@ mysqld_ld_library_path=
 flush_caches=0
 numa_interleave=0
 wsrep_on=0
+simulate=0
```

Comment:
Alright, so mysqld_safe does some non-trivial parsing indeed. Your approach seem to be reasonable, though I believe we can simplify it a bit. Please find my suggestions inline.